fiftyone/core/ontology.py

```
@@ -0,0 +1,249 @@
+                    type="str",
+                    component="dropdown",
+                    values=["front", "rear", "driver_side", "passenger_side"],
+                    when=[When(WhenOperator.EQUALS, field="damage_present", value=True)],
+                ),
+            ],
+        )
+    """
+
+    _TYPE = OntologyType.ANNOTATION_ONTOLOGY.value
+
+    def __init__(
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)